### PR TITLE
Preset Kool.yml customization

### PR DESCRIPTION
**Pull request by danielsuguimoto**
Branch based on #220

### CODE REVIEW

File: cmd/parser/fake_yml.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package parser
```
Comment (fabriciojs) [inline cmd/parser/fake_yml.go:1]: haven't we started adding `build ignore` tags to fake files?

File: cmd/parser/yml.go [hunk 2]
```diff
@@ -38,6 +51,17 @@ func ParseKoolYaml(filePath string) (parsed *KoolYaml, err error) {
 	return
 }
 
+// Parse decodes the target kool.yml
+func (y *KoolYaml) Parse(filePath string) (err error) {
+	var parsed *KoolYaml
+	if parsed, err = ParseKoolYaml(filePath); err != nil {
+		return
```
Comment (fabriciojs) [inline cmd/parser/yml.go:58]: As Codecov complained, isn't there a test we can run with a bad formatted file, so we get this error covering hence the line?

**Review by fabriciojs — COMMENTED** [review]
Looking very nice! Just a couple minor comments.

Although looking at all the changes here, I am starting to think: maybe we need to move the templates themselves off this repository? But that is a thought for the future...

File: cmd/parser/fake_yml.go (hunk 1 above)
Comment (danielsuguimoto) [reply cmd/parser/fake_yml.go:1]: we have a separated issue for that #224

File: cmd/parser/yml.go (hunk 2 above)
Comment (danielsuguimoto) [reply cmd/parser/yml.go:58]: @fabriciojs yep, done!

**Review by fabriciojs — APPROVED** [review]

**Pull request merged**